src/commands/node.mjs

```diff
@@ -526,7 +526,33 @@ export class NodeCommand extends BaseCommand {
         title: 'Fetch platform software into network nodes',
         task:
           async (ctx, task) => {
-            return self.fetchPlatformSoftware(ctx, task, self.platformInstaller)
+            if (argv[flags.localBuildPath.name] !== '') {
+              const config = ctx.config
+              self.logger.debug('no need to fetch, use local build jar files')
+              const localDataLibBuildPath = argv[flags.localBuildPath.name] + '/hedera-node/data'
```

Comment:
instead of `+` to concat path, we need to use `path.join()` so that it also works on windows

Reply:
Please also check that `configManager.getFlag(flags.localBuildPath)` also works and returns the same value of `argv[flags.localBuildPath.name]`

Reply:
Updated